lib/adapter/freshheads/CopyFilesToBuildDirAdapter.ts

```diff
@@ -4,19 +4,25 @@ import { BuilderConfig } from '../../Builder';
 import path from 'path';
 import { validateIfRequiredModuleIsInstalled } from '../../utility/moduleHelper';
 
-// @todo use types from @types/copy-webpack-plugin instead (cannot export CopyPattern right now)
-type CopyPattern = {
-    from: {
-        glob: string;
-    };
-    context?: string;
+// @todo use types from @types/copy-webpack-plugin instead (cannot export StringPattern / ObjectPattern right now)
```

Comment:
In nieuwe versie van de types worden deze nog steeds niet geexporteerd ...

Reply:
irritant :|